src/main/java/io/stargate/sgv2/jsonapi/api/v1/response/MapToRestResult.java

```diff
@@ -0,0 +1,28 @@
+package io.stargate.sgv2.jsonapi.api.v1.response;
+
+import io.stargate.sgv2.jsonapi.api.model.command.CommandResult;
+import java.util.Optional;
+import org.jboss.resteasy.reactive.RestResponse;
+
+public class MapToRestResult {
+  /**
+   * Maps CommandResult to RestResponse. Except for few selective errors, all errors are mapped to
+   * http status 200. In case of 401, 500, 502 and 504 response is sent with appropriate status
+   * code.
+   *
+   * @param commandResult
+   * @return
+   */
+  public static RestResponse map(CommandResult commandResult) {
```

Comment:
this can also be  a method  `CommandResult`

Reply:
Will do.